### PR TITLE
CHAD-8808 Qubino Thermostat:

### PR DESCRIPTION
**Pull request by greens**
Only display the currently configured mode's setpoint

### CODE REVIEW

**Comment by greens** [comment]
@dsobierajsk could you help me out by generating a presentation to correspond to these changes?

File: drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat-cooling.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  - name: "thermostatMode"
+    title: "Thermostat Mode"
+    description: "This setting allows to change mode of the device. Remember to unpair to pair device again after change."
+    preferenceType: enumeration
+    required: true
+    definition:
+      options:
+        0: Heat
+        1: Cool
+      default: "0"
```
Comment (dsobierajsk) [inline drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat-cooling.yml:34]: The default value should be 1, shouldn't it?

Comment (greens) [reply drivers/SmartThings/zwave-thermostat/profiles/qubino-flush-thermostat-cooling.yml:34]: good point!

**Review by dsobierajsk — APPROVED** [review]

File: drivers/SmartThings/zwave-thermostat/src/qubino-flush-thermostat/init.lua [hunk 2]
```diff
@@ -86,9 +86,11 @@ local function configuration_report(driver, device, cmd)
     if configuration_value == 1 then
       device:set_field(CONFIGURED_MODE, COOL_MODE, {persist = true})
       table.insert(supported_modes, capabilities.thermostatMode.thermostatMode.cool.NAME)
+      device:try_update_metadata({profile = "qubino-flush-thermostat-cooling"})
     elseif configuration_value == 0 then
       device:set_field(CONFIGURED_MODE, HEAT_MODE, {persist = true})
       table.insert(supported_modes, capabilities.thermostatMode.thermostatMode.heat.NAME)
+      device:try_update_metadata({profile = "qubino-flush-thermostat"})
```
Comment (KKlimczukS) [inline drivers/SmartThings/zwave-thermostat/src/qubino-flush-thermostat/init.lua:93]: @greens shouldn't "qubino-flush-thermostat" be renamed to "qubino-flush-thermostat-heating" ?

Comment (greens) [reply drivers/SmartThings/zwave-thermostat/src/qubino-flush-thermostat/init.lua:93]: I was just going to leave it that way so we would know which was the default.

**Pull request merged**